test/manifest.js

```diff
@@ -0,0 +1,201 @@
+import path from 'path';
+import test from 'ava';
+import pEvent from 'p-event';
+import rev from '../';
+import createFile from './fixtures';
+
+const manifestFixture = './fixtures/test.manifest-fixture.json';
+const manifestFixturePath = path.join(__dirname, manifestFixture);
+const manifestFixtureRelative = path.join('test', manifestFixture);
+
+test('should build a rev manifest file', async t => {
+	const stream = rev.manifest();
+	const data = pEvent(stream, 'data');
+
+	stream.write(createFile({
+		path: 'unicorn-d41d8cd98f.css',
+		revOrigPath: 'unicorn.css'
+	}));
+	stream.end();
```

Comment:
In all the tests that only write one file, you can pass it to `stream.end()` instead of `stream.write()` as a shortcut.

```js
stream.end(createFile({
  path: 'unicorn-d41d8cd98f.css',
  revOrigPath: 'unicorn.css'
}));
```